### ISSUE TITLE
[JS] Retrieve function not working as expected

### ISSUE THREAD

**Issue by Champkinz** [firebase/genkit#212]
**Describe the bug**
When i try to follow the [documentation](https://firebase.google.com/docs/genkit/plugins/firebase#usage) on the Firebase genkit plugin, i get the below error

The code to use:
```
const docs = await retrieve({
  retriever: yourRetrieverRef,
  query: 'look for something',
  config: { limit: 5 },
});
```

The error:
> Cannot read properties of undefined (reading 'where')
> TypeError: Cannot read properties of undefined (reading 'where')
>     at /Users/abc/abc/abc/abc/ai-functions/node_modules/@genkit-ai/firebase/lib/firestoreRetriever.js:110:35
>     at Generator.next (<anonymous>)
>     at fulfilled (/Users/abc/abc/abc/abc/ai-functions/node_modules/@genkit-ai/firebase/lib/firestoreRetriever.js:33:24)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**To Reproduce**
- Define a Firestore retriever as shown on [example](https://firebase.google.com/docs/genkit/plugins/firebase).
- Use the below code and run a flow or a firebase cloud function call
```
const docs = await retrieve({
  retriever: yourRetrieverRef,
  query: 'look for something',
  config: { limit: 5 },
});
```

**Expected behavior**
- To get the documents that is related for my query

**Runtime (please complete the following information):**
 - OS: [MacOS]
 - Version [0.5.0]

**Node version**
 -  v20.12.2

**Additional context**
So the correct way to use this by passing the below code, instead of whats shown on the documentation
```
const docs = await retrieve({
  retriever: yourRetrieverRef,
  query: 'look for something',
  options: { limit: 5 },
});
```

i just want to know if the documentation is wrong in this scenario or am i doing something wrong ?

**Comment by ssbushi** [comment]
Hi @Champkinz 

Thanks for catching this bug -- you're right, this is a bug in the documentation, the correct field name for retriever options should be `options`.  Do you still run into this issue after applying the above fix?

**Comment by pavelgj** [comment]
The fix is now released in 0.5.1, please give it another try.

**Comment by Champkinz** [comment]
> Hi @Champkinz
> 
> Thanks for catching this bug -- you're right, this is a bug in the documentation, the correct field name for retriever options should be `options`. Do you still run into this issue after applying the above fix?

Yes, when i add options it works fine.

Thank you for the release.
